<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1287</width>
    <height>893</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StarrTracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1281</width>
      <height>841</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="horizontalSpacing">
      <number>14</number>
     </property>
     <property name="verticalSpacing">
      <number>45</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="layout_MapView" stretch="64,656">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="layout_LiveRocketView">
         <property name="leftMargin">
          <number>16</number>
         </property>
         <property name="rightMargin">
          <number>16</number>
         </property>
         <item>
          <widget class="QLabel" name="Label_LiveRocketView">
           <property name="font">
            <font>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Live Rocket View</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>  Map View Widget Goes Here                                                    </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="layout_Graph">
       <property name="spacing">
        <number>14</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="layout_x_axis">
         <item>
          <widget class="QLabel" name="x_axis">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>17</pointsize>
            </font>
           </property>
           <property name="text">
            <string>  X Position (ddmm.mmmm)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCustomPlot" name="graph_x" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>250</height>
            </size>
           </property>
           <property name="whatsThis">
            <string>Latitude</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layout_y_axis">
         <item>
          <widget class="QLabel" name="y_axis">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>17</pointsize>
            </font>
           </property>
           <property name="text">
            <string>  Y Position (ddmm.mmmm)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCustomPlot" name="graph_y" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="whatsThis">
            <string>Longitude
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layout_altitude">
         <item>
          <widget class="QLabel" name="altitude">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>17</pointsize>
            </font>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Altitude&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>  Altitude (1 unit = 1m)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCustomPlot" name="graph_alt" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="whatsThis">
            <string>Altitude</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="layout_Settings" stretch="0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" name="layout_Serial">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Serial Connections</string>
           </property>
           <widget class="QComboBox" name="portSelect">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>30</y>
              <width>79</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QPushButton" name="serialConnectionButton">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>80</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
           <widget class="QPushButton" name="serialTerminationButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>60</y>
              <width>80</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>41</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Port </string>
            </property>
           </widget>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>110</y>
              <width>291</width>
              <height>221</height>
             </rect>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Baud rate</string>
            </property>
           </widget>
           <widget class="QLabel" name="gpsType">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>291</width>
              <height>17</height>
             </rect>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>DataGPS</string>
            </property>
           </widget>
           <widget class="QComboBox" name="baudSelect">
            <property name="geometry">
             <rect>
              <x>220</x>
              <y>30</y>
              <width>86</width>
              <height>25</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>110</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>300</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>38400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>57600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>128000</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>256000</string>
             </property>
            </item>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>110</height>
          </size>
         </property>
         <property name="title">
          <string>Read Local Data</string>
         </property>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>41</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Path</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="pathBrowser">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>30</y>
            <width>291</width>
            <height>41</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="serialLocalConnectionButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>80</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
         <widget class="QPushButton" name="serialLocalTerminationButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>80</y>
            <width>80</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1287</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit_Program"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_StarrTracker"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_StarrTracker">
   <property name="text">
    <string>About StarrTracker</string>
   </property>
  </action>
  <action name="actionExit_Program">
   <property name="text">
    <string>Exit Program</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">src/graph/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
